generators/python/src/fern_python/generators/sdk/client_generator/endpoint_function_generator.py

```diff
@@ -1165,6 +1173,12 @@ def _get_headers_for_endpoint(
         if endpoint.idempotent:
             ir_headers += idempotency_headers
 
+        if endpoint.request_body is not None:
+            unioned_value = endpoint.request_body.get_as_union()
+            if isinstance(unioned_value, ir_types.InlinedRequestBody):
```

Comment:
can we check the discriminant here instead? `unioned_value.type === "inlined..."`

Reply:
resolved.